src/pubsub/ua_pubsub_manager.c

```diff
@@ -4,6 +4,8 @@
  *
  * Copyright (c) 2017-2019 Fraunhofer IOSB (Author: Andreas Ebner)
  * Copyright (c) 2018 Fraunhofer IOSB (Author: Julius Pfrommer)
+ * Copyright (c) 2020 Yannick Wallerer, Siemens AG
+ * Copyright (c) 2020 Thomas Fischer, Siemens AG
```

Comment:
The file seems currently not updated by this PR. Therefore, the copyright notice is needed.